packages/wonder-blocks-modal/components/two-column-modal.js

```diff
@@ -27,6 +25,15 @@ type Props = {
      * clicks too.)
      */
     onClickCloseButton: () => void,
+
+    /** The footer to show under the main column's content. */
+    footer?: React.Node,
+
+    /** The main column's content. */
+    contents: React.Node,
```

Comment:
I'm not sure this column actually is more "main" than the other. There's [some discussion][1] in the spec about this, and it's a bit in-flux. I proposed `leftColumn` and `rightColumn` to avoid the implied hierarchy, but then Kevin pointed out that maybe there _is_ some natural hierarchy because only the main column is present in the mobile design, then I pointed out that maybe that's not the intent of the design spec? We meant to ask Tabitha, but I'm not sure where we got with that. I've added a Zeplin comment now (51).

In any case, if we do decide to go with the hierarchical naming, I think this is `content` in the spec rather than `contents`.

[1]: https://docs.google.com/a/khanacademy.org/document/d/1__cz2bHvZGWtBIcf_W0IGpGtgu1HKgHIjS9mmYsjbxk/edit?disco=AAAABw4OXrQ

Reply:
Hmm - ok! So when I saw that the "sidebar" went away in the mobile view that definitely indicated to me that it was secondary. But yeah, it's also not always a sidebar as it goes to the top in the mobile view. @tyong any thoughts on what we should be naming these things?